### ISSUE TITLE
Prevent scrolling on question change

### ISSUE THREAD

**Issue by jheathco** [surveyjs/survey-creator#996]
### Are you requesting a feature, reporting a bug or ask a question?

Requesting a feature, or asking how to perform a specific behavior

### What is the current behavior?

Oftentimes when going to the next question, the page will scroll

### What is the expected behavior?

Scrolling shouldn't happen

**Comment by andrewtelnov** [comment]
@jheathco If you are talking about SurveyJS Library, then we scroll to elements in two cases: on changing a page (next/prev), it is optional, and on showing the first question that has an error on atempting go to the next page or complete the survey.
If you are talking about SurveyJS Designer, then we are scrolling to a new added element.
Could you please describe us steps that we should perform to reproduce the issue?

Thank you,
Andrew

**Comment by jheathco** [comment]
Correct, on switching page - where is that functionality described so we can turn it off/

Thanks!

**Comment by jheathco** [comment]
Apologies, I realize I likely opened this for the wrong project.
